### ISSUE TITLE
Computation of accuracy metrics of BRAKER prediction

### ISSUE THREAD

**Issue by ConcettaDe4** [Gaius-Augustus/BRAKER#132]
Hi!

I ran BRAKER on my species to annotate for the first time the genome. 
I would like to compute the accuracy metrics of my predictions in term of sensitivity and specificity. 
I have some questions about how to obtain the accuracy values.
I was wondering if it is correct to consider as accuracy of my predictions the accuracy measures (i.e. sensitivity and specificity) from the Augustus files firsttest.out and secondtest.out. 
Otherwise, is it better to compare the predicted genes by BRAKER with the test genes obtained with AUGUSTUS using a tool like cuffcompare?

Thank you for your help!

Concetta

**Comment by KatharinaHoff** [comment]
The values in firsttest.out and secondtest.out are measured as follows:

The initial training gene set for AUGUSTUS is created using genes predicted by GeneMark that were filtered by evidence, i.e. every intron must be supported by at least one intron hint from RNA-Seq/protein data. Single exon genes may be random, or if particular protein evidence (CDSpart hints) are available, may be evidence supported.

That gene set is split into a test and a training set. The training set is further split into two sets, never mind. The test set is not using during training and optimization, i.e. it remains as an independent test set. After training/optimization, AUGUSTUS predicts genes in this test set and then produces the accuracy values.

The numbers give a good estimate about training and optimization success. They are not taken against a reference gold standard. If you need to evaluate against an existing gene set, predict genes in the genome and use a tool such as eval to evaluate. (braker.pl can actually do that automatically if you provide the reference gene set in the correct format, but might be easier for you to do this separately.)

Katharina

**Comment by ConcettaDe4** [comment]
Thank you for your answer.

I have one more question. 

Unfortunately, I do not have an existing gene set to evaluate my predictions. 

In this case, to have the accuracy metrics of BRAKER prediction can I compare the final prediction in the whole genome with the genes in the testing set, that AUGUSTUS did not use during training and optimization?

Thank you.

Concetta

**Comment by KatharinaHoff** [comment]
You could in theory do that, but we don't provide a set of scripts for this. BRAKER will only report ab initio prediction accuracy (directly measured on the genbank format holdout test set file)
